OTOnexus/Classes/Capture/OTOCaptureView.swift

```diff
@@ -337,31 +337,29 @@ class OTOCaptureView: UIView {
             DispatchQueue.global(qos: DispatchQoS.QoSClass.default).async {
                 var videoConnection : AVCaptureConnection?
                 
-                if let connections = stillOutput.connections as? [AVCaptureConnection] {
```

Comment:
This whole thing is just indented after removing the `if let` statement on line 340